### ISSUE TITLE
Shipyards should group ship choices

### ISSUE THREAD

**Issue by dmcbride** [plainblack/Lacuna-Server-Open#360]
We could tag ships with different types, and those types would appear in the UI so that users don't have to scroll through 20+ ship types to find the one they want, and it would provide an implicit clue as to what kind of ship it is when they open the type.

For example, attack ships (sweepers, seekers, snarks), defense ships (sweepers, fighters), cargo ships (dory, cargo, freighter, smuggler, hulk, scow?), waste ships (scow), platforms (mining, terra, gas giant) - not sure where to put excav, and probably have forgotten some off the top of my head.

**Comment by ashtraynaut** [comment]
imho, this could work:

* colonizing ships
colony ship - short range colony ship - terraform ship - gas giant platform ship - stake - SS Hull
* industrial ships
excavator - mining platform ship - scow - fissure sealer - supply pod
* reconaissance ships
probe - surveyor - scanner
* electronic warfare
detonator - bleeder - placebo - thud
* covert ops
spy pod - spy shuttle - smuggler ship

**Comment by Imzogelmo** [comment]
I like both of your lists, and I assume the ships don't have to fit in only one filter, correct?

Basically a combination of your two lists would cover most situations, maybe with some minor tweaks. 
* Trade
* Waste
* Attacks
* Defense
* Support
* Spies
* Exploration
* Platforms

**Comment by dmcbride** [comment]
Buildings don't fit in a single list, I don't see why ships should, either.  In fact, my example showed sweepers in both attack and defense, so I was anticipating multiple tasks for some ships.  Smugglers would be in trade and spy, for example (I'd try to keep all categories either singular or plural, but that's a minor nit).

And, for the record, this is HGMNY's idea, not mine, I just created the issue to track it because I thought it a good enough idea to track.

**Comment by Imzogelmo** [comment]
Well, it's a very good idea, kudos to HGMNY

**Comment by 1vasari** [comment]
:+1:

**Comment by Harlinator** [comment]
Would be nice to see this in Spaceport view also :)

**Comment by ashtraynaut** [comment]
yes i have proposed this list as an idea of how to sort ships into a list
which will give you a good overview of what you have.
right now, ships are sorted by ID in your spaceport. so any new ship will
be put at the end of the list.

that can be a very annoying thing if you are looking for a certain ship.

in the Trade Ministry it is even more complex because ships are not sorted
by ID but it is more or less sorted by speed, cargo hold size and tier.
so right now, if i want to know if i have any snarks on a base, i click a
random, non-allied, colony and see what ships will appear in the send-list.

it's a solution, though not yet THE solution ;)
a list where we can sort ships based on type, tier or speed would be great.

On Sat, Jun 20, 2015 at 3:12 PM, Harlinator <notifications@github.com>
wrote:

> Would be nice to see this in Spaceport view also :)
>
> Sent from my iPhone
>
>
> > On Jun 17, 2015, at 20:01, Nathan McCallum <notifications@github.com>
> wrote:
> >
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plainblack/Lacuna-Server-Open/issues/360#issuecomment-113766661>
> .
>



-- 

 *Ƹ̵̡Ӝ̵̨̄Ʒ **There is no one like you*Ƹ̵̡Ӝ̵̨̄Ʒ

**Comment by 1vasari** [comment]
Again, :+1:!

**Comment by Zephyr101** [comment]
Whatever, as long as we can get rid of 80 pages of 1 ship would be an improvement!  You are doing a great job 1vasari, thank you.

**Comment by zhodson** [comment]
I think something similar to the way ship are shown in the TM tab could work
